python/interpret-core/interpret/visual/_inline.py

```diff
@@ -243,6 +244,8 @@ def render(explanation, id_str=None, default_key=-1, detected_envs=None, js_url=
         or "azuresynapse" in detected_envs
     ):
         display(HTML(init_js + body_js))
+    elif "streamlit" in detected_envs:
```

Comment:
My main question is whether we can avoid adding the streamlit dependency for all installations.  Does it still work if the import streamlit.components.v1 statement is put inside this elif "streamlit" block?  Presumably if "streamlit" is detected in the environment the streamlit dependency is installed?

Reply:
Yes, you're right, it works fine.
I'm going to create a new PR and close this one because I didn't do it properly.